### PR TITLE
Balsamic v9 SV callers - Fix #3346

### LINKED ISSUE #3346: Add SV/CNV callers from merged SVDB VCF for cancer cases
BALSAMIC uses the following variant caller attribute in the INFO of VCF file for SVs and CNVs
    manta
    dellysv
    cnvkit
    ascat
    dellycnv
It would be nice to show these with list of SVs and CNVs showing callers supporting the variant.
(tiddit will be added soon)

### PR DESCRIPTION
**Pull request by dnil**
This PR adds a functionality or fixes a bug.

Balsamic v9 SV caller tag constants.

<details>
<summary>Testing on cg-vm1 server (Clinical Genomics Stockholm)</summary>

**Prepare for testing**
1. Make sure the PR is pushed and available on [Docker Hub](https://hub.docker.com/repository/docker/clinicalgenomics/scout-server-stage)
1. `ssh <USER.NAME>@cg-vm1.scilifelab.se`
1. `sudo -iu hiseq.clinical`
1. `ssh localhost`
1. (optional) Find out which scout branch is currently deployed on cg-vm1: `podman ps`
1. Stop the service with current deployed branch: `systemctl --user stop scout.target`
1. Start the scout service with the branch to test: `systemctl --user start scout@<this_branch>`
1. Make sure the branch is deployed: `systemctl --user status scout.target`
</details>

**How to test**:
1. how to test it, possibly with real cases/data

**Expected outcome**:
The functionality should be working
Take a screenshot and attach or copy/paste the output.

**Review:**
- [ ] code approved by
- [ ] tests executed by

### CODE REVIEW

File: scout/constants/__init__.py [hunk 1]
```diff
@@ -142,8 +142,13 @@
         {"id": "freebayes", "name": "Freebayes"},
     ],
     "cancer_sv": [
-        {"id": "manta", "name": "Manta"},
         {"id": "gatk", "name": "GATK"},
+        {"id": "manta", "name": "Manta"},
```
Comment (northwestwitch) [inline scout/constants/__init__.py:146]: 👍🏻

**Review by northwestwitch — APPROVED** [review]
Looks good, have you checked that the tags correspond ti the names in the VCFs?

**Comment by dnil** [comment]
> Looks good, have you checked that the tags correspond ti the names in the VCFs?

I think that by now is a quite obvious no. I just copied from the Issue text; seemed reasonable and similar to the ones already present and in the demo files.

**Comment by dnil** [comment]
I guess I'll just merge this anyway - it doesn't hurt, may not have to be changed, and if so it's easy enough.

**Pull request merged**